test/framework/tinkerbell.go

```diff
@@ -191,8 +193,13 @@ func (t *Tinkerbell) WithNewWorkerNodeGroup(name string, workerNodeGroup *Worker
 	panic("Not implemented for Tinkerbell yet")
 }
 
-func envVarForImage(os OS, kubeVersion anywherev1.KubernetesVersion) string {
+func envVarForImage(os OS, kubeVersion anywherev1.KubernetesVersion, rtos ...bool) string {
 	imageEnvVar := fmt.Sprintf("T_TINKERBELL_IMAGE_%s_%s", strings.ToUpper(strings.ReplaceAll(string(os), "-", "_")), strings.ReplaceAll(string(kubeVersion), ".", "_"))
+	if len(rtos) > 0 {
+		if rtos[0] {
```

Comment:
nit: can we combine these?
```suggestion
	if len(rtos) > 0 && rtos[0] {
			imageEnvVar = fmt.Sprintf("%s_RTOS", imageEnvVar)
		}
```